docs/source/guide/distributed-partition.rst

```diff
@@ -69,6 +69,33 @@ second dictionary contains the mapping for each edge type.
     orig_node_emb = th.zeros(node_emb.shape, dtype=node_emb.dtype)
     orig_node_emb[node_map] = node_emb
 
+As a counterpart of ``return_mapping=True`` in :func:`~dgl.distributed.partition_graph`,
+below 2 arguments are provided for distributed partition pipeline in ``dispatch_data.py``.
```

Comment:
Try to avoid passive tone as much as possible, e.g.,

```
As a counterpart of ``return_mapping=True`` in :func:`~dgl.distributed.partition_graph`,
the distributed partitioning pipeline <give a link> provides two arguments in ``dispatch_data.py``
to save the original node/edge IDs to disk.
```